infrastructure/deploy_box_instance_data.sh

```diff
@@ -0,0 +1,56 @@
+# This script can be used as the instance-user-data.sh script for AWS
+# instances to configure a new instance to be used as a deploy box.
+
+# To create a new deploy box (there only ever needs to be one at a
+# time.), there are few steps to follow:
+#  - Make sure there is a deploy VPC. If there isn't create one so the deploy box is isolated.
+#  - Make sure the deploy VPC has a security group. If it doesn't create one and allow ingress
+#    TCP connections on port 22 from 0.0.0.0/0
+#  - Create a new EC2 instance with the following settings:
```

Comment:
Create it where? Via console or TF? Include URL here. Maybe a picture. Or couldn't this be TF managed as well?